### PR TITLE
Feature/add confirmed cases to actueel

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
On the homepage with the 'actuele situatie in Nederland' there needs to be a 'positief geteste mensen' tile. with the information about the confirmed cases with a time chart and a positive percentage of confirmed cases tested by the GGD.

### CODE REVIEW

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Review by Denniswegereef — APPROVED** [review]
Looking good 👍

File: packages/app/src/components/data-driven-text/components/metric.tsx [hunk 1]
```diff
@@ -39,22 +39,19 @@ export function Metric<T extends DataKeys, K = DataFile<T>>({
         | DifferenceDecimal)
     : undefined;
 
-  assert(
```
Comment (Denniswegereef) [inline packages/app/src/components/data-driven-text/components/metric.tsx:42]: I think this assert still needs to be here?

File: packages/app/src/components/data-driven-text/components/metric.tsx [hunk 2]
```diff
@@ -39,22 +39,19 @@ export function Metric<T extends DataKeys, K = DataFile<T>>({
...
-    isDefined(differenceValue),
-    `Missing value for difference:${differenceKey}`
-  );
-
```
Comment (Denniswegereef) [inline packages/app/src/components/data-driven-text/components/metric.tsx:46]: I think this assert still needs to be here?

Comment (VWSCoronaDashboard1) [inline packages/app/src/components/data-driven-text/components/metric.tsx:46]: This assert doesn't need to be here in this form. But I think that it does need AN assert.
But one that checks if `differenceKey` is defined, and in that case the `differenceValue` needs to exist

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: packages/app/src/components/data-driven-text/components/metric.tsx [hunk 3]
```diff
@@ -39,22 +39,26 @@ export function Metric<T extends DataKeys, K = DataFile<T>>({
         | DifferenceDecimal)
     : undefined;
 
-  assert(
-    isDefined(differenceValue),
-    `Missing value for difference:${differenceKey}`
-  );
+  if (isUndefined(differenceKey)) {
```
Comment (VWSCoronaDashboard16) [inline packages/app/src/components/data-driven-text/components/metric.tsx:42]: Nit: `!isDefined` saves an extra import ;)

**Review by VWSCoronaDashboard16 — APPROVED** [review]
Looks good! Two tiny nitpicks

**Pull request merged**